src/main/java/Deadline.java

```diff
@@ -0,0 +1,15 @@
+public class Deadline extends Task {
+    protected String deadline;
+
+
+    public Deadline(String description, String tasktype, String deadline) {
+        super(description, tasktype);
```

Comment:
The tasktype argument is unnecessary for all constructors of Task's subclasses since they are fixed.